### DISCUSSION TITLE
StreamX Data Studio 功能特性讨论

### DISCUSSION THREAD

**Discussion by Al-assad** [apache/streampark#565] [Ideas]
### 关于 StreamX Data Studio 功能特性的讨论

   对于 StreamX 的 Data Studio 功能，主要用于满足交互式的数据开发场景，应该包含 SQL 智能补全、Meta 元数据集成、数据血缘、数据预览、在线调试日志等功能。
  
   我糊了一个 Data Studio 的概念原型（支持简单的在线交互）：[https://www.figma.com/file/IuLLEvfkyVhBB4SCjeaF1J/Data-Studio?node-id=0%3A1](https://www.figma.com/file/IuLLEvfkyVhBB4SCjeaF1J/Data-Studio?node-id=0%3A1)
  
   主要包含 3 个新增的模块：
* Data Studio：提供数据开发 IDE、支持元数据自动补全、数据预览、调试日志；
* Data Source：提供元数据数据集成、catalog 挂载、元数据浏览；
* Cluster Center：提供交互式查询引擎集群（如 Flink Session、Trino 等）的部署、管理；

**Comment by wolfboys** [comment]
非常棒的特性,这是streamx接下来的重点工作.结合社区开发者的力量公共完成这一激动人心的特性.

**Comment by Narcasserun** [comment]
+1
